apps/dashboard/components/navbar.tsx

```diff
@@ -0,0 +1,131 @@
+import { Button } from "@unkey/ui";
+import { cn } from "@unkey/ui/src/lib/utils";
+import { MoreHorizontal } from "lucide-react";
+import React from "react";
+
+type BaseProps = React.PropsWithChildren<React.HTMLAttributes<HTMLElement>>;
+
+type LinkProps = {
+  href: string;
+  active?: boolean;
+  isIdentifier?: boolean;
+  isLast?: boolean;
+  className?: string;
+  children: React.ReactNode;
+};
+
+type BreadcrumbsComponent = React.ForwardRefExoticComponent<
+  BaseProps & React.RefAttributes<HTMLElement> & { icon: React.ReactNode }
+> & {
+  Link: React.ForwardRefExoticComponent<LinkProps & React.RefAttributes<HTMLAnchorElement>>;
+  Ellipsis: React.ForwardRefExoticComponent<React.HTMLAttributes<HTMLLIElement>>;
+};
+
+interface GlobalNavbarComponent
+  extends React.ForwardRefExoticComponent<BaseProps & React.RefAttributes<HTMLElement>> {
+  Actions: React.ForwardRefExoticComponent<BaseProps & React.RefAttributes<HTMLDivElement>>;
+  Breadcrumbs: BreadcrumbsComponent;
+}
+
+export const Navbar = React.forwardRef<HTMLElement, BaseProps>(
+  ({ children, className, ...props }, ref) => (
+    <nav
+      ref={ref}
+      className={cn(
+        //TODO: "bg-background" used here as a temporary fix. In the design its "#FFF", but it breaks our background color. We'll replace this later.
+        "w-full p-4 border-b border-gray-4 bg-background justify-between flex",
+        className,
+      )}
+      {...props}
+    >
+      {children}
+    </nav>
+  ),
+) as GlobalNavbarComponent;
+
+Navbar.Actions = React.forwardRef<HTMLDivElement, React.HTMLAttributes<HTMLDivElement>>(
+  ({ className, ...props }, ref) => (
+    <div ref={ref} className={cn("flex items-center gap-3", className)} {...props}>
+      {props.children}
+    </div>
+  ),
+);
+
+const Breadcrumbs = React.forwardRef<HTMLElement, BaseProps & { icon: React.ReactNode }>(
+  ({ children, className, icon, ...props }, ref) => {
+    const childrenArray = React.Children.toArray(children);
+    return (
+      <nav ref={ref} aria-label="breadcrumb" className={cn("flex", className)} {...props}>
+        <ol className="flex items-center gap-3">
```

Comment:
```suggestion
        <ol className="flex items-center gap-3 list-none">
```

Have a feeling this will show list numbers if used in markdown file as examples

Reply:
I think we won't need that coz that component should live in `ui` package. This will be used only in Dashboard.

Reply:
@ogzhanolguncu